.github/workflows/buf.yaml

```diff
@@ -0,0 +1,48 @@
+name: buf
+on:
+  push:
+    paths:
+      - 'proto/**'
+jobs:
+  lint:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: bufbuild/buf-setup-action@v0.3.1
+        if: success()
+        with:
+          version: 0.56.0
+      - uses: bufbuild/buf-lint-action@v0.3.0
+        if: success()
+        with:
+          buf_token: ${{ secrets.BUF_TOKEN }}
+  breaking:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: bufbuild/buf-setup-action@v0.3.1
+        if: success()
+        with:
+          version: 0.56.0
+      - uses: bufbuild/buf-breaking-action@v0.4.0
+        if: success()
+        with:
+          against: buf.build/bufbuild/core
+          buf_token: ${{ secrets.BUF_TOKEN }}
```

Comment:
We should allow this to fail, right?

Reply:
Not anymore - we ignore unstable packages